nvmw.bat

```diff
@@ -75,27 +81,44 @@ exit /b 0
 :install
 setlocal
 
+set NODE_TYPE=node

```

Comment:
After really use in two days, keep all version into special folder look better. 
e.g.: `node/v0.11.10` better than `v0.11.10`